include/fem-dem/vans_assemblers.h

```diff
@@ -171,11 +171,82 @@ class GLSVansAssemblerPFI : public ParticleFluidAssemblerBase<dim>
    */
   virtual void
   calculate_particle_fluid_interactions(
-    NavierStokesScratchData<dim> &scratch_data) override;
+    NavierStokesScratchData<dim> &scratch_data,
+    SimulationParameters<dim> &) override;
+
+  std::shared_ptr<SimulationControl> simulation_control;
+  Parameters::PhysicalProperties     physical_properties;
+};
+
+/**
+ * @brief Class that assembles the drag force using Rong model for the
+ * VANS equations.
```

Comment:
Would be a good idea to add the equation for the Rong model here.

Reply:
done